tmva/sofie/inc/TMVA/ROperator_Max.hxx

```diff
@@ -0,0 +1,87 @@
+#ifndef TMVA_SOFIE_ROPERATOR_MAX
+#define TMVA_SOFIE_ROPERATOR_MAX
+
+#include "TMVA/SOFIE_common.hxx"
+#include "TMVA/ROperator.hxx"
+#include "TMVA/RModel.hxx"
+
+#include <sstream>
+
+namespace TMVA{
+namespace Experimental{
+namespace SOFIE{
+
+template <typename T>
+class ROperator_Max final : public ROperator
+{
+
+private:
+
+   std::string fNX1;
```

Comment:
The MAX operator can have as input a list of tensors of arbitrary size. Here we are limited to 2, so it is like a binary operator. 
You should have a `std::vector<std::string>` defining  the input tensor names